homeassistant/components/tellduslive/entry.py

```diff
@@ -33,25 +32,16 @@ def __init__(self, client, device_id):
         """Initialize the entity."""
         self._id = device_id
         self._client = client
-        self._async_unsub_dispatcher_connect = None
 
     async def async_added_to_hass(self):
         """Call when entity is added to hass."""
         _LOGGER.debug("Created device %s", self)
-        self._async_unsub_dispatcher_connect = async_dispatcher_connect(
-            self.hass, SIGNAL_UPDATE_ENTITY, self._update_callback
+        self.async_on_remove(
+            async_dispatcher_connect(
+                self.hass, SIGNAL_UPDATE_ENTITY, self.async_write_ha_state
```

Comment:
This signal is not specific to the config entry to it will update all entities when it fires, but this is an existing problem, and I wanted to keep this change small since I cannot test it